### PR TITLE
JDKIM compilation problems

### PR DESCRIPTION
**Pull request by chibenwa**
Review only last commit https://github.com/linagora/james-jdkim/commit/0b4f4b476346c8ce41461c5796cdb29f5169668e

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -22,9 +22,10 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>org.apache.james</groupId>
-        <artifactId>james-project</artifactId>
-        <version>3.0.0-beta5-SNAPSHOT</version>
+        <groupId>org.apache</groupId>
+        <artifactId>apache</artifactId>
```
Comment (rouazana) [inline pom.xml]: org.apache.apache?

File: pom.xml [hunk 2]
```diff
@@ -22,9 +22,10 @@
...
+        <version>18</version>
```
Comment (rouazana) [inline pom.xml]: why?

**Comment by chibenwa** [comment]
1. You comment an already approuved changeset
2. Can not remember, it was done several months before....

File: pom.xml (hunk 1 above)
Comment (chibenwa) [reply pom.xml]: screenshot from 2017-05-03 18-13-08

Taken from james-project

File: pom.xml (hunk 2 above)
Comment (chibenwa) [reply pom.xml]: This is the version we are using in James project

**Review by rouazana — APPROVED** [review]
Yep sorry, I saw your main comment after.

**Review by quynhn — APPROVED** [review]

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**